schemes/utilities/physics_tendency_updaters.F90

```diff
@@ -117,4 +118,29 @@ subroutine apply_tendency_of_air_temperature_run(nz, t_tend, temp, dtdT_total,
 
    end subroutine apply_tendency_of_air_temperature_run
 
+   !> \section arg_table_apply_constituent_tendencies_run Argument Table
+   !!! \htmlinclude apply_constituent_tendencies_run.html
+   subroutine apply_constituent_tendencies_run(nz, const_tend, const, dt, errcode, errmsg)
+      ! Dummy arguments
+      integer,            intent(in)    :: nz                 ! Num vertical layers
+      real(kind_phys),    intent(inout) :: const_tend(:,:,:)  ! constituent tendency array
+      real(kind_phys),    intent(inout) :: const(:,:,:)       ! constituent state array
+      real(kind_phys),    intent(in)    :: dt                 ! physics time step
+      integer,            intent(out)   :: errcode
+      character(len=512), intent(out)   :: errmsg
+
+      ! Local variables
+      integer :: klev
+
+      errcode = 0
+      errmsg = ''
+
+      do klev = 1, nz
+         const(:, klev, :) = const(:, klev, :) + (const_tend(:, klev, :) * dt)
+      end do
+
+      const_tend = 0._kind_phys
```

Comment:
I just realized that this zeroing out step is something we should probably do for the other tendency updaters too.  Could you possibly add it to them as well?